index.html

```diff
@@ -61,9 +60,152 @@ <h1>I’m Alexandra.   Glad   to  see      you!</h1>
                 </div>
               </div>
 
+             
+
+              <section class="mywork">
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectone.png">
+                            <h3 class="ptitle">Tonic</h3>
+                        </div>
+                        <div class="p-details">
+                            <ul>
+                                <li>CANOPY</li>
+                                
+                                <li>Back End Dev</li>
+                               
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <div class="lower_part">
+                            <ul class="projectstack">
+                                <li><a>HTML</a></li>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </section>
```

Comment:
Great job on implementing the cards! However, there are a few areas that require improvement to align with the design and improve the overall user experience.

-  Firstly, consider replacing the `<section>` tag with an `<article>` tag for the card element. This will ensure proper semantic structure and accurately represent the purpose of the card within the page.
-  Additionally, the appearance of the cards does not match the design. Pay attention to aligning the cards and their elements, including font styles, spacing, and border-radius, to closely resemble the design. Make the necessary adjustments to achieve a consistent and visually appealing result in the card.
-  Lastly, the gap between the cards should match the design specifications. Ensure that the spacing between two cards is consistent and follows the design guidelines.

| Your | Design|
| --- | --- |
| ![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/55840999/e533856e-07e6-49e7-94f8-b4541e17d475) | ![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/55840999/382959e0-4de7-4b72-9cbf-81e05934eb8b) |

By addressing these points, you will bring your implementation closer to the desired design, improving the overall aesthetics and user experience. Keep up the good work!